inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -709,8 +709,16 @@ std::shared_ptr<ngraph::Node> V10Parser::XmlDeserializer::createNode(
         }
         ngraphNode->set_arguments(inputs);
         XmlDeserializer visitor(node, weights, opsets);
-        if (ngraphNode->visit_attributes(visitor))
+        if (ngraphNode->visit_attributes(visitor)) {
             ngraphNode->constructor_validate_and_infer_types();
+        }
+
+        // To be sure that all default values will be initialized:
+        ngraphNode = ngraphNode->clone_with_new_inputs(ngraphNode->input_values());
+
+        if (auto subGraph = std::dynamic_pointer_cast<ngraph::op::util::SubGraphOp>(ngraphNode)) {
```

Comment:
please add a comment here with mention this ticket 36145 and a short description (like ctor of Loop and TI do not call validate_and infer_function)
auto -> const auto&

Reply:
done